brandingbrand/flagship
# refactor(fscomponents): FLAGSHIP-68 update to function component

### Ticket
https://jira.brandingbrand.com/browse/FLAGSHIP-68

### Description
TabbedContainer used to be a class-based component. This PR is to update it to a function component.

### Test
1. run `yarn && yarn build:storybook && yarn dev:storybook`
2. navigate to *TabbedContainer* and then *basic usage*
3. click on the tabs and observe the change

![Screen Shot 2020-01-17 at 3 31 58 PM](https://user-images.githubusercontent.com/51350319/72644310-b96e7180-393e-11ea-85c9-ea9c184ddbc4.png)
